src/js/components/DateInput/DateInput.js

```diff
@@ -102,7 +111,8 @@ const DateInput = forwardRef(
                 // clicking an edge date removes it
                 else if (range) normalizedValue = [nextValue, nextValue];
                 else normalizedValue = nextValue;
-                if (schema) setTextValue(valueToText(normalizedValue, schema));
+                if (schema)
+                  setTextValue(valueToText(normalizedValue, schema, range));
```

Comment:
I'm not seeing a range arg in the utils.js valueToText. Am I missing something?

Reply:
good catch, fixed